feed page ui done #23

ThapaVinay opened the pull request:
![Screenshot from 2023-07-27 16-23-05](https://github.com/upes-open/OSoC-Swayam-Bazaar/assets/97795121/fe755b60-a4dc-43c2-8e84-c4ec950118bb)
![Screenshot from 2023-07-27 16-22-52](https://github.com/upes-open/OSoC-Swayam-Bazaar/assets/97795121/779171d7-66d4-4efd-9f00-b837f7017bd4)

prathamkandari commented:
@ThapaVinay these are two different pages. I've given you the points for the feedback page. But you have to change the feed page UI then only I'll be able to give points

ThapaVinay commented:
Updated feed page also added navbar

![Screenshot from 2023-07-28 12-42-18](https://github.com/upes-open/OSoC-Swayam-Bazaar/assets/97795121/88d3ed56-cdc2-4572-8265-ce4e205ad9d5)
![Screenshot from 2023-07-28 12-37-55](https://github.com/upes-open/OSoC-Swayam-Bazaar/assets/97795121/8a07f3f7-f72c-4cc7-bbef-31092ffce543)

prathamkandari commented:
Well done @ThapaVinay
